### ISSUE TITLE
Dolby: Focus styles not clear enough in hero (breadcrumbs and CTA button)

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#199]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/dolbyaudioexperience/
Focus color Is invert (I think) which should be enough to make it show up, but in this case with image background, that decision looks like it isn't working as well as we hoped. 

(50% of the CTA button looks ok, but the breadcrumb is hardly visible at all)
